### PR TITLE
Adding in shortcut key for stations popover

### PR DESCRIPTION
**Pull request by tylerthetiger**
This PR sets Cntrl-R to toggle the stations popover to make keyboard navigation easier as requested in Issue #648 .

### CODE REVIEW

**Comment by brotherJ4mes** [comment]
Thanks tylerthetiger!  

I just went to test this out but then remembered I installed pithos thru flatpak and I don't have a good grasp on how flatpak actually works.....  Is it safe to just (sudo) modify the python code installed at the crazy long flatpak path that I find with `locate`?  

If not, I can totally just be patient and wait for the merge to trickle down.

**Comment by tylerthetiger** [comment]
I'm not that familiar with flatpak, but I'd imagine you'd be fine to just modify the 2 python files from that directory.

File: pithos/pithos.py [hunk 1]
```diff
@@ -474,6 +474,10 @@ def init_actions(self, app):
         app.add_accelerator('<Primary>d', 'win.bookmark', None)
         action.connect('activate', self.bookmark_song)
 
+        action = Gio.SimpleAction.new('station-popover', None)
+        self.add_action(action)
+        app.add_accelerator('<Primary>r', 'win.station-popover', None)
+        action.connect('activate', self.stations_popover.make_visible)
```
Comment (TingPing) [inline pithos/pithos.py:480]: Add a newline after this.

File: pithos/StationsPopover.py [hunk 2]
```diff
@@ -140,6 +140,11 @@ def clear(self):
         for row in self.listbox.get_children():
             row.destroy()
 
+    def make_visible(self,*ignore):
+        if self.props.visible:
+            self.hide()
+        else:
+            self.show_all()
```
Comment (TingPing) [inline pithos/StationsPopover.py:147]: Newline here too.

File: pithos/StationsPopover.py [hunk 3]
```diff
@@ -140,6 +140,11 @@ def clear(self):
         for row in self.listbox.get_children():
             row.destroy()
 
+    def make_visible(self,*ignore):
```
Comment (TingPing) [inline pithos/StationsPopover.py:143]: Space between function args.

Comment (TingPing) [inline pithos/StationsPopover.py:143]: `make_visible` isn't really accurate. Probably `toggle_visibility`

**Comment by TingPing** [comment]
Other than the nitpicks seems fine. I dunno about the shortcut choice but I guess its ok.

File: pithos/pithos.py [hunk 4]
```diff
@@ -474,6 +474,10 @@ def init_actions(self, app):
         app.add_accelerator('<Primary>d', 'win.bookmark', None)
         action.connect('activate', self.bookmark_song)
 
+        action = Gio.SimpleAction.new('station-popover', None)
+        self.add_action(action)
+        app.add_accelerator('<Primary>r', 'win.station-popover', None)
```
Comment (TingPing) [inline pithos/pithos.py:479]: The action name can be improved to be descriptive, maybe `win.toggle-station-popover`

**Comment by brotherJ4mes** [comment]
@tylerthetiger I gave up with trying to mod the flatpak install and instead just cloned your fork and installed from source--Working great so far.  

Thanks so much for this great feature add!  ...It's so nice to be able to switch up the music without taking my hands off the keyboard.

**Pull request closed without merging**